OMPython/__init__.py

```diff
@@ -62,6 +62,21 @@
 import pyparsing
 from distutils import spawn
 
+# The following import are added by Sudeep
+import numpy as np
+import pandas as pd
+import csv
+
+import matplotlib.pyplot as plt
```

Comment:
probably it can be dropped, as it's only used in a [very thin wrapper](https://github.com/OpenModelica/OMPython/pull/7/files#diff-a1709e22cd56663c5abde760173fd676R953) around plt.plot. Same with the other dependencies, pd is only used a couple of times to return some pd dataframes (maybe the user should do that instead if desired)´, and numpy is used only twice (although one is an interp, so may be difficult to replace).